dftimewolf/lib/processors/turbinia_gcp.py

```diff
@@ -43,79 +44,69 @@ def __init__(self, state, critical=False):
       critical (Optional[bool]): True if the module is critical, which causes
           the entire recipe to fail if the module encounters an error.
     """
-    super(TurbiniaProcessor, self).__init__(state, critical=critical)
+    super(TurbiniaProcessorBase, self).__init__(state, critical=critical)
     self._output_path = None
     self.client = None
-    self.disk_name = None
     self.instance = None
     self.project = None
+    self.run_all_jobs = None
+    self.sketch_id = None
     self.turbinia_region = None
     self.turbinia_zone = None
-    self.sketch_id = None
-    self.run_all_jobs = None
 
   # pylint: disable=arguments-differ
-  def SetUp(self, disk_name, project, turbinia_zone, sketch_id, run_all_jobs):
+  def TurbiniaSetUp(self, project, turbinia_zone, sketch_id, run_all_jobs):
     """Sets up the object attributes.
 
     Args:
-      disk_name (str): name of the disk to process.
       project (str): name of the GPC project containing the disk to process.
```

Comment:
This is the project where Turbinia lives? Admittedly, it contains the disks to process, but it would be clearer IMO to explicitly assume that this is the case.

Reply:
Yeah, I'll fix this up a bit, thanks.